src/tests/suite/daos_rebuild_ec.c

```diff
@@ -164,7 +164,7 @@ rebuild_ec_internal(void **state, bool parity, int type)
 
 	print_message("Insert %d kv record in object "DF_OID"\n",
 		      KEY_NR, DP_OID(oid));
-
+D_PRINT("[RYON] %s:%d [%s()] > \n", __FILE__, __LINE__, __FUNCTION__);
```

Comment:
(style)  unnecessary whitespace before a quoted newline
(style)  __func__ should be used instead of gcc specific __FUNCTION__